src/CMakeLists.txt

```diff
@@ -120,6 +120,10 @@ if (HAVE_MPI)
   INCLUDE_DIRECTORIES("${qmcpack_SOURCE_DIR}/external_codes/mpi_wrapper")
 endif()
 
+if (HAVE_LIBBOOST)
```

Comment:
Change my mind a bit. MA is not part of boost yet. It must have its own macro if needed.
At the moment, MA is shipped within the code and used by default. We don't need to use a macro.
@markdewing what do you think?